### DISCUSSION TITLE
Do you plan to make an advanced ToDos system with calendar?

### DISCUSSION THREAD

**Discussion by Terkyz** [streetwriters/notesnook#3714] [Q&A]
In the absence of ToDo cross-platform open source programs/apps it really bothers me to have to use programs like TickTick to organize my tasks. The main problem is that it's not open source and then the fragmentation between services, so I was wondering, is it planned in the future to add advanced ToDo features in Notesnook? I mean, sure, I can create a notebook with reminders and use the task boxes but it's not the same, I'd like to have it all in one place.

It's probably out of the scope of Notesnook but I'm still curious if something like this would be implemented in the future.

**Comment by thecodrr** [comment]
> It's probably out of the scope of Notesnook 

There you have it. It _is_ out of scope.

However, feel free to specify exactly what you require because there are plans to allow setting reminders on specific tasks in a Task List. There is no plan to include a Calendar though.

**Reply by Nalci** [reply]
I would like a bit more advanced task system.
- I miss the possibility to mark some task with priority (1,2,3...)
- as I wrote in [#1794](https://github.com/streetwriters/notesnook/issues/1794#issuecomment-1937710332) I would like to write some details to a task but without cluttering too much.

I'm currently using todoist but would love to switch fully to notesnook.
